vaas-app/src/vaas/adminext/templates/forms/priority.html

```diff
@@ -1,19 +1,31 @@
 {% spaceless %}{% include 'django/forms/widgets/select.html' %}{% endspaceless %}
 <script>
+
+    function getChosenClusters() {
+            let clusterQuery = ""
```

Comment:
We used underscore naming convention for variables earlier - so 
"cluster_query"